### PR TITLE
Wallet template tutorial

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
Added tutorial for wallet template which helps in connecting to different wallets.

### CODE REVIEW

File: docs/tutorials/t-05.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+   Now you can select your preferred network and connect to the wallet as per your choice.
+   <img src="./t-05/assets/address_selected.png" height="200" title="Account Selected" />
+
+   You can now interact with the algorand blockchain, by sending algos, or call an application.
+   <img src="./t-05/assets/algo_transaction.png" height="200" title="Send Transaction" />
+
+   You can also update the address to which algos are sent, the algo amount or the application ID by configuring it in `src/constants`.
+
+   ```js
```
Comment (vuvoth) [inline docs/tutorials/t-05.md:51]:
```suggestion
   ```ts
```

File: docs/tutorials/t-05.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+   Migrate to wallet directory<br/>
+   `cd wallet`
+
+ #### 2. Environment Setup
+   Install all dependencies in your environment:<br/>
+   ` yarn install` OR `npm install`
+
+   Now, run the app for local development. It will automatically update when there are any changes.<br/>
+   `yarn run dev`
```
Comment (vuvoth) [inline docs/tutorials/t-05.md:38]:
```suggestion
   `yarn run dev` or `npm run dev`
```

Comment (Megha-Dev-19) [reply docs/tutorials/t-05.md:38]: oh yes, thanks for reminding

File: docs/tutorials/t-05.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+This is a Web Application created using Vue through which one can connect to Algorand Blockchain and perform transaction (for eg: send Algo or call application) by connecting to your algorand wallet, select different accounts available on different networks such as MainNet, TestNet, BetaNet.
+
+### Steps
+
+ #### 1. Development
+   The source code for this tutorial can be found on GitHub. Start by cloning the repository:<br/>
+   `git clone https://github.com/scale-it/algo-builder-templates.git`
+
+   Change into the project directory:<br/>
+   `cd algo-builder-templates`
```
Comment (robert-zaremba) [inline docs/tutorials/t-05.md:21]:
```suggestion
#### 1. Setup
The source code for this tutorial can be found on GitHub. Start by cloning the repository

    git clone https://github.com/scale-it/algo-builder-templates.git
    cd algo-builder-templates/wallet
```

File: docs/tutorials/t-05.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+   The repository contains the following templates:<br/>
```
Comment (robert-zaremba) [inline docs/tutorials/t-05.md:23]: why there is this additional indentation?

File: docs/tutorials/t-05.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+   **htlc** : This is a descriptive example explaining how HTLC contracts work with Algo-Builder.<br/>
+   **monitor-payments**: This is a nodejs app which monitors payment transactions to a particular address.<br/>
+   **shop**: This demonstrates the usage of a payment widget to trigger a purchase by connecting to different wallets.<br/>
+   **wallet**: This helps in connecting to different wallets.<br/>
+   **warcraft**: This is a descriptive example explaining how Escrow contract work with Algo-Builder.<br/>
+
+   Migrate to wallet directory:<br/>
+   `cd wallet`
+
```
Comment (robert-zaremba) [inline docs/tutorials/t-05.md:32]:
```suggestion
```

File: docs/tutorials/t-05.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+ #### 2. Environment Setup
+   Install all dependencies in your environment:<br/>
+   ` yarn install` or `npm install`
+
+   Now, run the app for local development. It will automatically update when there are any changes.<br/>
+   `yarn run dev` or `npm run dev`
+
+   After successfully running, your browser screen will look similar to this.
+   <img src="./t-05/assets/homescreen.png" height="200" title="Home Page" />
```
Comment (robert-zaremba) [inline docs/tutorials/t-05.md:41]: please rename this file to `t-wallet.md` and create `t-wallet` directory for additional assets

File: docs/tutorials/t-05.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
```
Comment (robert-zaremba) [inline docs/tutorials/t-05.md:42]:
```suggestion

#### 3. Using the Wallet Widget
```

File: docs/tutorials/t-05.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+   You can also update the address to which algos are sent, the algo amount or the application ID by configuring it in `src/constants`.
+
+   ```ts
+   export const toAddress = "WK73JQQGCRQCZ5GLQHCXXB6OXIGF6SD22F5P7HPXF5PNH23YUUALUMKOZ4";
+   export const amount: number = 10e6;
+   export const applicationId: number = 189;
+   ```
+
+   And finally you can disconnect your chosen wallet.
```
Comment (robert-zaremba) [inline docs/tutorials/t-05.md:57]: Add one more step explaining how user can add style, and add a screenshot with a link to an example (the SigmaWallet) showing how the wallet can look like with proper styling (only take a screenshot of the wallet widget)

File: docs/tutorials/t-05.md (hunk 6 above)
Comment (Megha-Dev-19) [reply docs/tutorials/t-05.md]: updated the names

File: docs/tutorials/t-05.md (hunk 8 above)
Comment (Megha-Dev-19) [reply docs/tutorials/t-05.md]: @robert-zaremba  added the new step, Could you take a look at it.?

**Comment by robert-zaremba** [comment]
Why we don't see a network selection in [styled_wallet.png](https://github.com/scale-it/algo-builder/pull/738/files#diff-f7688ad39e4ee6311bb318b2a7e5edff8a1caf0c6b0795c840c1d75578ca5023) ?

**Comment by Megha-Dev-19** [comment]
> Why we don't see a network selection in [styled_wallet.png](https://github.com/scale-it/algo-builder/pull/738/files#diff-f7688ad39e4ee6311bb318b2a7e5edff8a1caf0c6b0795c840c1d75578ca5023) ?

Will add one more screenshot for network and wallet selection.

File: docs/tutorials/t-wallet.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
+The purpose of the tutorial is a demonstration of Algorand Wallet Widget integrated with the [Algo Builder Web](https://github.com/scale-it/algo-builder/tree/master/packages/web) package, written in Vue JS. The widget connects to major Algorand wallets (Algo Signer, Wallet Connect, MyAlgoWallet), sign and send transactions constructed with Algo SDK or Algo Builder framework.
```
Comment (ratik21) [inline docs/tutorials/t-wallet.md:1]:
```suggestion
The purpose of this tutorial is a demonstration of Algorand Wallet Widget integrated with the [@algo-builder/web](https://github.com/scale-it/algo-builder/tree/master/packages/web) package, written in Vue JS. The widget connects to major Algorand wallets (Algo Signer, Wallet Connect, MyAlgoWallet), sign and send transactions constructed with Algo SDK or Algo Builder framework.
```

File: docs/tutorials/t-wallet.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+- IDE, e.g. [VSCode](https://code.visualstudio.com/download)<br/>
+- Other dependencies are listed in [package.json](https://github.com/scale-it/algo-builder-templates/blob/master/wallet/package.json). Install them with yarn install or npm install.<br/>
+
+### Background
+
+Algorand ecosystem lacks a maintained Vue JS component connected to main Algorand web wallets.
+
+[Algo Builder](https://algobuilder.dev/) is one of the main major frameworks for smart contract development and integrations. We took a responsibility to provide the community a Vue JS wallet widget, well integrated with the Algo Builder framework and Algo SDK.
+
+In this tutorial we create a basic Vue JS web application which connects web wallets and perform simple transaction (for eg: send Algo or call application). With the wallet widget a user can:
```
Comment (ratik21) [inline docs/tutorials/t-wallet.md:17]:
```suggestion
In this tutorial we will create a basic Vue JS web application which connects web wallets and perform a simple transaction (for eg: send Algo or call application). With the wallet widget a user can:
```

File: docs/tutorials/t-wallet.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+      </select>
+     </div>
+  </div>
+```
+
+For eg: [Sigma Wallet](https://github.com/scale-it/algobuilder-msig) has a styled wallet widget using [Antd](https://www.antdv.com/) library.
+<img src="./t-wallet/assets/select_network.png" height="100" title="Select Network" />
+<img src="./t-wallet/assets/styled_wallet.png" height="200" title="Styled Wallet Widget" />
+
+The wallet component and the whole web page can easily be customized to fulfil your UX design requirements along with providing required functionality.
```
Comment (ratik21) [inline docs/tutorials/t-wallet.md:111]:
```suggestion
The wallet component above and the whole web page can easily be customized to fulfil your UX design requirements along with providing the required functionality.
```

File: docs/tutorials/t-wallet.md [hunk 12]
```diff
@@ -0,0 +1,111 @@
+The purpose of this tutorial is a demonstration of Algorand Wallet Widget integrated with the [@algo-builder/web](https://github.com/scale-it/algo-builder/tree/master/packages/web) package, written in Vue JS. The widget connects to major Algorand wallets (Algo Signer, Wallet Connect, MyAlgoWallet), sign and send transactions constructed with Algo SDK or Algo Builder framework.
```
Comment (robert-zaremba) [inline docs/tutorials/t-wallet.md:1]:
```suggestion
# Algo Builder Wallet Widget 

The purpose of this tutorial is a demonstration of the Algo Builder Wallet Widget integrated with the [@algo-builder/web](https://github.com/scale-it/algo-builder/tree/master/packages/web) package, written in Vue JS. The widget connects to major Algorand wallets (Algo Signer, Wallet Connect, MyAlgoWallet), sign and send transactions constructed with Algo SDK or Algo Builder framework.
```

File: docs/tutorials/t-wallet.md [hunk 13]
```diff
@@ -0,0 +1,113 @@
...
+- [NodeJS](https://nodejs.org/en/download/) > v14.11.0<br/>
+- JavaScript package manager - either [YARN](https://classic.yarnpkg.com/en/docs/install) > v3.1.0 or NPM which comes with NodeJS.<br/>
+- IDE, e.g. [VSCode](https://code.visualstudio.com/download)<br/>
+- Other dependencies are listed in [package.json](https://github.com/scale-it/algo-builder-templates/blob/master/wallet/package.json). Install them with yarn install or npm install.<br/>
+
+### Background
+
+Algorand ecosystem lacks a maintained Vue JS component connected to main Algorand web wallets.
+
+[Algo Builder](https://algobuilder.dev/) is one of the main major frameworks for smart contract development and integrations. We took a responsibility to provide the community a Vue JS wallet widget, well integrated with the Algo Builder framework and Algo SDK.
```
Comment (robert-zaremba) [inline docs/tutorials/t-wallet.md:17]:
```suggestion
[Algo Builder](https://algobuilder.dev/) is one of the major frameworks for smart contract development and integrations. We took a responsibility to provide the community a Vue JS wallet widget, well integrated with the Algo Builder framework and Algo SDK.
```

File: docs/tutorials/t-wallet.md [hunk 14]
```diff
@@ -0,0 +1,113 @@
...
+
+    git clone https://github.com/scale-it/algo-builder-templates.git
+    cd algo-builder-templates/wallet
+
+The repository contains the following templates:<br/>
+**htlc** : This is a descriptive example explaining how HTLC contracts work with Algo-Builder.<br/>
+**monitor-payments**: This is a nodejs app which monitors payment transactions to a particular address.<br/>
+**shop**: This demonstrates the usage of a payment widget to trigger a purchase by connecting to different wallets.<br/>
+**wallet**: This helps in connecting to different wallets.<br/>
+**warcraft**: This is a descriptive example explaining how Escrow contract work with Algo-Builder.<br/>
```
Comment (robert-zaremba) [inline docs/tutorials/t-wallet.md:39]:
```suggestion
**htlc** : a descriptive example explaining how HTLC contracts work with Algo-Builder.<br/>
**monitor-payments**: a nodejs app which monitors payment transactions to a particular address.<br/>
**shop**: demonstrates a payment widget connected to different web wallets.<br/>
**wallet**: (this tutorial) Algo Builder Wallet Wideget tutorial.<br/>
**warcraft**: a descriptive example explaining how to create an escrow account with Algo Builder.<br/>
```

File: docs/tutorials/t-wallet.md [hunk 15]
```diff
@@ -0,0 +1,113 @@
...
+
+After successfully running, your browser screen will look similar to this.
+<img src="./t-wallet/assets/homescreen.png" height="200" title="Home Page" />
+
+#### 3. Using the Wallet Widget
+
+Now you can select your preferred network and connect to the wallet as per your choice.
+<img src="./t-wallet/assets/address_selected.png" height="200" title="Account Selected" />
+
+You can now interact with the algorand blockchain, by sending algos, or call an application.
```
Comment (robert-zaremba) [inline docs/tutorials/t-wallet.md:57]:
```suggestion
You can now interact with the algorand blockchain (send algos, or call an application):
```

File: docs/tutorials/t-wallet.md [hunk 16]
```diff
@@ -0,0 +1,113 @@
...
+export const toAddress = "WK73JQQGCRQCZ5GLQHCXXB6OXIGF6SD22F5P7HPXF5PNH23YUUALUMKOZ4";
+export const amount: number = 10e6;
+export const applicationId: number = 189;
+```
+
+And finally you can disconnect your chosen wallet.
+
+#### 3. Style the Wallet Widget
+
+You can also style your wallet widget with own custom styles or using an external library by editing the `App.vue` file to integrate your styled classes or components.
```
Comment (robert-zaremba) [inline docs/tutorials/t-wallet.md:72]:
```suggestion
You can also style your wallet widget using your own custom styles or using an external library by editing the `App.vue` file:
```

**Review by robert-zaremba — APPROVED** [review]
left few language improvements

**Pull request merged**